addons/web/static/src/legacy/js/core/session.js

```diff
@@ -6,6 +6,7 @@ var concurrency = require('web.concurrency');
 var core = require('web.core');
 var mixins = require('web.mixins');
 var utils = require('web.utils');
+const { session } = require('@web/session');
```

Comment:
Hello, the `im_livechat.external_lib` bundle is including the legacy file, but not the new file, so this import is actually not working there.

You might want to also check `enterprise/project_timesheet_synchro/views/timesheet.html` because it is also importing it.